ddtrace/contrib/wsgi/wsgi.py

```diff
@@ -1,19 +1,13 @@
-import sys
+from typing import TYPE_CHECKING
 
-from ddtrace.internal.compat import PY2
 
+if TYPE_CHECKING:
+    from typing import Callable
+    from typing import Iterable
 
-if PY2:
-    import exceptions
+    from ddtrace import Tracer
 
-    generatorExit = exceptions.GeneratorExit
```

Comment:
ugh, this is unfortunately in the public api 😢 

We have to leave this in place by importing it back from `_utils`